src/detectors/__MOCKS__/Golang/BackendGoDetector.mock.ts

```diff
@@ -0,0 +1,24 @@
+import { IProjectComponentDetector } from '../../IProjectComponentDetector';
+import {
+  LanguageAtPath,
+  ProjectComponent,
+  ProjectComponentFramework,
+  ProjectComponentPlatform,
+  ProjectComponentType,
+} from '../../../model';
+import { injectable } from 'inversify';
+
+@injectable()
+export class BackendJavaDetector implements IProjectComponentDetector {
```

Comment:
```suggestion
export class BackendGolangDetector implements IProjectComponentDetector {
```

Reply:
my bad, working on vim with no typescript plugin/helper does this. updated